charts/rancher-monitoring/v0.0.7/charts/prometheus/templates/secrets.yaml

```
@@ -25,4 +25,20 @@ metadata:
   name: {{ template "app.fullname" . }}-additional-alertmanager-configs
 stringData:
   additional-alertmanager-configs.yaml: {{ template "additional-alertmanager-configs.yaml" . }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{- if .Values.thanos.enabled }} 
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "app.thanos.fullname" . }}
+  labels:
+    app: {{ template "app.name" . }}
+    chart: {{ template "app.version" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+stringData:
+  thanos.yaml: |-
+{{ toYaml .Values.thanos.objectConfig | indent 6 }}
+{{-end }}
```

I think these needs to be:
```suggestion
{{- end }}
```

---

Why?

---

Nvm. I read the diff wrong. Fixing.